src/Control/Monad/Aff.js

```diff
@@ -320,3 +320,11 @@ exports._tailRecM = function (isLeft, f, a) {
     }(a);
   };
 };
+
+
+var avar = require("../Control.Monad.Aff.AVar/foreign.js");
```

Comment:
I think `pulp test` just runs it in node (no bundling), so it would work. But I don't think psc-bundle will follow this reference, and you'll end up with a stray `require` for it.